bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -239,11 +238,25 @@ <h1 class="large">{{_('Check Your Plugins')}}</h1>
     </div>
     <p><a href="#top">{{_('Back to Top')}}</a></p>
 </section>
+
+<!-- dislayed to nonFx users -->
+<div class="not-supported">
+  <header id="main-feature">
+      <h1 class="large">{{_('Check Your Plugins')}}</h1>
+      <p class="preamble">{{_('Keeping your plugins up to date helps Firefox run safely and smoothly.')}}</p>
+  </header>
+
+  <section>
+    <p>
```

Comment:
Should this `<p>` be inside the l10n tag, else if the locale does not have the string translated, we would just put outputting an empty tag?